addons/crm/views/crm_recurring_plan_views.xml

```diff
@@ -1,41 +1,55 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<odoo><data>
+<odoo>
+  <data>
 
     <record id="crm_recurring_plan_view_tree" model="ir.ui.view">
-        <field name="name">crm.recurring.plan.view.tree</field>
-        <field name="model">crm.recurring.plan</field>
-        <field name="arch" type="xml">
-            <tree editable="bottom">
-                <field name="sequence" widget="handle"/>
-                <field name="name"/>
-                <field name="number_of_months"/>
-            </tree>
-        </field>
+      <field name="name">crm.recurring.plan.view.tree</field>
+      <field name="model">crm.recurring.plan</field>
+      <field name="arch" type="xml">
+        <tree editable="bottom">
+          <field name="sequence" widget="handle"/>
+          <field name="name"/>
+          <field name="number_of_months"/>
+          <field name="start_date"/>
+          <field name="end_date"/>
```

Comment:
These fields don't seem to exist for that model. Recurring plans can only exist in terms of months. You would need to add modifications to the python code to support this.

Feel free to submit a change to use dates if you feel it's somehow relevant but I would encourage you motivate the change first. Personally I don't see a usecase for that.